documentation/modules/creating-network-mapping.adoc

```diff
@@ -17,13 +22,11 @@ You can create one or more network mappings by using the {ocp} web console to ma
 
 . In the {ocp} web console, click *Migration* -> *NetworkMaps for virtualization*.
 . Click *Create NetworkMap*.
-. Complete the following fields:
+. Specify the following fields:
 
-* *Name*: Enter a name to display in the network mappings list.
+* *Name*: Enter a name to display in the network mappings list. For example, `Default network map` or `Empty network mapping`.
```

Comment:
I'd skip the "Empty network mapping" part, see my previous comment

Reply:
Removed sentence. it was only there to mention "empty network mapping".